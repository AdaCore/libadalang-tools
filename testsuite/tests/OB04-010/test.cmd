contract_coverage *.ads *.adb
