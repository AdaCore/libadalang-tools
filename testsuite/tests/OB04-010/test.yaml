description:
    Tests basic features of the contract_coverage tool.

driver: contract_coverage
