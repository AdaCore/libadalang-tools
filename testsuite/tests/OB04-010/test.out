Contract coverage for package Empty: 100%
Contract coverage for package P1: 100%
Contract coverage for package P2: 0%
Contract coverage for package P3: 50%

